orc8r/cloud/go/obsidian/server/server.go

```diff
@@ -43,6 +46,18 @@ func Start() {
 	e.Use(CollectStats)
 	e.Use(middleware.Recover())
 
+	// If the feature is toggled on, serve the dynamically generated Swagger
+	// Spec instead of the static asset.
```

Comment:
Can remove this comment -- it's clear from the documentation of the config value

Instead, replace it with a description of how we're doing something a little funky -- implicitly overriding the static endpoint with a dynamic endpoint, and how that functionality isn't explicitly documented

I know we're planning on changing this in the next PR but it's good practice to keep up high quality, because you never know how long something might end up in the codebase if e.g. you suddenly need to pivot what you're working on